src/gui/sharedialog.h

```diff
@@ -88,13 +88,14 @@ private slots:
     QByteArray _numericFileId;
     QString _privateLinkUrl;
     ShareDialogStartPage _startPage;
-    ShareManager *_manager;
+    ShareManager *_manager = nullptr;
 
-    QList<ShareLinkWidget*> _linkWidgetList;
-    ShareLinkWidget* _emptyShareLinkWidget;
-    ShareUserGroupWidget *_userGroupWidget;
-    QProgressIndicator *_progressIndicator;
+    QList<ShareLinkWidget*> _linkWidgetList = {};
```

Comment:
This is not needed, default ctor being invoked anyway.

Reply:
Right, I'm starting to get paranoid in the code ^^

Reply:
(already strange it was so in the constructor)